nanoFramework.Logging.Syslog/Facility.cs

```diff
@@ -0,0 +1,41 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+using System;
```

Comment:
You'll call me picky... But it does require an extra enter after the // and before the using.